service/dap/server.go

```diff
@@ -1191,7 +1191,7 @@ func (s *Server) convertVariableWithOpts(v *proc.Variable, qualifiedNameOrExpr s
 	}
 	value = api.ConvertVar(v).SinglelineString()
 	if v.Unreadable != nil {
-		return
+		return value, variablesReference
```

Comment:
why?

Reply:
https://tour.golang.org/basics/7

"Naked return statements should be used only in short functions, as with the example shown here. They can harm readability in longer functions."

Reply:
Ah, thanks for the pointer. If readability is at stake here, then probably "return value, 0" would be even better.